src/lib/components/profile-picture.svelte

```diff
@@ -94,8 +94,8 @@
       <div class="my-2 w-full border-[0.5px] border-skin-text-highlight" />
       <a
         href="/api/authentication/logout"
-        class="default-transition underline decoration-primary-100 hover:decoration-transparent"
-        >Sign out</a
+        class="default-transition decoration-primary-100 hover:decoration-transparent hover:text-red-600 text-red-500"
```

Comment:
I think we shouldn't use some random colors in here, and stick with the eddiehub ci, what do you think?

Reply:
I think the same also we should stick with eddiehub ci

Reply:
Okay, I will revert these changes and align with the EddieHub CLI color scheme.